packages/amplify-provider-awscloudformation/src/aws-utils/aws-pinpoint.js

```diff
@@ -39,7 +39,7 @@ async function getConfiguredPinpointClient(context, category, action, envName) {
     // ignore missing config
   }
   category = category || 'missing';
-  action = action || 'missing';
+  action = action || ['missing'];
```

Comment:
Is this a part of this change?

Reply:
It's an unrelated fix for a minor bug I stumbled upon when working on this